config/key.yml

```diff
@@ -70,3 +70,10 @@ cyclemap:
   - { min_zoom: 14, max_zoom: 19, name: bicycle_shop, image: bicycle_shop.png }
   - { min_zoom: 14, max_zoom: 19, name: bicycle_parking, image: bicycle_parking.png }
   - { min_zoom: 16, max_zoom: 19, name: toilets, image: toilets.png }
+opnvkarte:
+  - { min_zoom: 6, name: rail, source: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADQAAAAECAIAAABOYIsRAAAAF0lEQVQoz2Nsa2tjGKyAiWEQg1HHkQsA43UBmpvJuFsAAAAASUVORK5CYII=" }
```

Comment:
Perhaps we could use svgs here for easier changeing?

`"data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg"><rect width="300" height="100" style="fill:rgb(0,0,255)" /></svg>"`

Reply:
I guess this will work for simple shapes.

Added svgs with various line widths for different zoom levels.